### PR TITLE
Feature/stij 123: Added form error messages.

### PR DESCRIPTION
**Pull request by meirege**

### CODE REVIEW

File: components/21-atoms/field-error/field-error.twig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{% if field_error%}
```
Comment (Jeroen005) [inline components/21-atoms/field-error/field-error.twig]: space

File: components/00-settings/_vars.scss [hunk 2]
```diff
@@ -172,6 +172,10 @@ $themes: (
     'field-border': 1px solid $color-light-gray,
     'field-placeholder': $color-light-gray,
 
+    'field-error-background': $color-red,
+    'field-error-color': $color-darkest-gray,
+    'field-error-border': #a31b00,
```
Comment (Jeroen005) [inline components/00-settings/_vars.scss]: Maybe we should use darken() or lighten() here.

Comment (meirege) [reply components/00-settings/_vars.scss]: I took the colors from gent_base drupal 7.x ...
These are used in the current site of Stad Ghent, so we should probably follow these defined colors, no?

File: components/31-molecules/fields/_fields.scss [hunk 3]
```diff
@@ -31,8 +31,9 @@ select {
   }
 
   &.error {
-    border-color: #a31b00;
-    background-color: #ff8770;
+    @include themify {
+      border-color: $field-error-border;
```
Comment (Jeroen005) [inline components/31-molecules/fields/_fields.scss]: border not border-color

**Pull request merged**